pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -140,7 +140,7 @@ type KubeProxyConfiguration struct {
 	ResourceContainer string `json:"resourceContainer"`
 	// udpIdleTimeout is how long an idle UDP connection will be kept open (e.g. '250ms', '2s').
 	// Must be greater than 0. Only applicable for proxyMode=userspace.
-	UDPIdleTimeout metav1.Duration `json:"udpTimeoutMilliseconds"`
+	UDPIdleTimeout metav1.Duration `json:"udpIdleTimeout"`
```

Comment:
this is noteworthy, though probably allowable because of alpha status

Reply:
as long as we have a good release note we're fine I guess